events.js

```diff
@@ -1,13 +1,89 @@
-let mobileBurger=document.querySelectorAll('.burger');
-let mobileMenu=document.querySelector('#mobile-menu');
-let menuItem=document.querySelectorAll('.menu-item').forEach(item=>{
-    item.addEventListener('click',check);
+const projects = [
+  {
+    title: 'Number around the internet',
+    img: 'images/SnapsPortfolio.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.',
+    listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveUrl: 'https://alexrs90.github.io/Portfolio/',
+    githubUrl: 'https://www.github.com/AlexRS90/Portfolio',
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'images/yogaDesktop.png',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.',
+    listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveUrl: 'https://alexrs90.github.io/Portfolio/',
+    githubUrl: 'https://www.github.com/AlexRS90/Portfolio',
+  },
+  {
+    title: 'School Web Page',
+    img: 'images/SnapsPortfolio.png',
+    description: 'Arrival entered an if drawing request. How daughters not promotion few knowledge contented. Yet winter law behind number stairs garret excuse. Minuter we natural conduct gravity if pointed oh no. Am immediate unwilling of attempted admitting disposing it. Handsome opinions on am at it ladyship. ',
+    listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveUrl: 'https://alexrs90.github.io/Portfolio/',
+    githubUrl: 'https://www.github.com/AlexRS90/Portfolio',
+  },
+];
```

Comment:
- [ ] Here you are asked to display pop-ups from the objects, meaning that each specific card, should have its own data from the array object. Adding projects in the future will be easy. you will only add it in one place - as another object in the projects array. When I open a pop-up, I'm getting only one object data, the Figma design is just there for illustrations purposes only (Only one object was used). You have three objects, let's use them all not just one.